Rework CrowdStrike guide to allow following a given platform's instructions top to bottom

iansltx opened the pull request:
For #33193.

iansltx committed:
Rework CrowdStrike article to allow following a given platform's instructions top to bottom

nonpunctual reviewed (commented):
I do not think we should change the form of the article. Thanks.

nonpunctual committed:
Standardize section titles for post-install scripts

Updated section titles for clarity regarding post-install scripts across macOS, Linux, and Windows.

nonpunctual reviewed (approved):
okiedokie.

nonpunctual committed:
Merge branch 'main' into crowdstrike-docs-cleanup
